src/components/themed/WalletListRow.js

```diff
@@ -1,80 +1,73 @@
 // @flow
 
-import type { EdgeCurrencyInfo } from 'edge-core-js'
 import * as React from 'react'
 import { ActivityIndicator, TouchableOpacity, View } from 'react-native'
+import FastImage from 'react-native-fast-image'
 
 import { Gradient } from '../../modules/UI/components/Gradient/Gradient.ui.js'
-import { connect } from '../../types/reactRedux.js'
-import { getCurrencyInfo } from '../../util/utils'
-import { type Theme, type ThemeProps, cacheStyles, useTheme, withTheme } from '../services/ThemeContext.js'
+import { memo } from '../../types/reactHooks.js'
+import { type Theme, cacheStyles, useTheme } from '../services/ThemeContext.js'
 import { EdgeText } from './EdgeText.js'
 
-type OwnProps = {
+type Props = {
   currencyCode: string,
+  exchangeRateText?: string | null,
+  exchangeRateType?: null | string, // Should be any of this values: 'neutral' | 'positive' | 'negative'
   children?: React.Node,
   icon?: React.Node,
+  iconUri?: string,
+  iconSize?: number,
```

Comment:
This should be called `iconSizeRem`. We are consistent with this naming, so it's clear which ones are in rem and which ones are in points. No suffix = points.